custodian/templates/slack_inspector.j2

```diff
@@ -0,0 +1,39 @@
+{
+   "attachments":[
+      {
+         "fallback":"Cloud Custodian Policy Violation",
+         "title":"Custodian",
+         "color":"{{ action['slack_msg_color']|default("danger") }}",
+         "fields":[
+            {
+               "title":"Resources",
+               "value":"{%- for resource in resources -%}
+                            {%- for findingResource in resource['resources'] -%}
+                                *{{ findingResource['details']['awsEcrContainerImage']['repositoryName'] }}: {{ findingResource['details']['awsEcrContainerImage']['imageTags']|join(',') }}*: {{ resource['title'] }} Severity {{ resource['severity'] }}\n
```

Comment:
Do you think it's worth saving `findingResource['details']['awsEcrContainerImage']` to a variable, to make this line shorter?

Reply:
You can set variables in templates like but I don't think that's going to make it more readable so I think it's ok as is.